Wikipedia/Code/NotificationsCenterCellViewModel+SheetActionExtensions.swift

```diff
@@ -4,66 +4,92 @@ import WMF
 
 extension NotificationsCenterCellViewModel {
     
-    enum SheetAction {
+    enum SheetAction: Equatable {
         case markAsReadOrUnread(SheetActionData)
         case custom(SheetActionData)
         case notificationSubscriptionSettings(SheetActionData)
+        
+        static func == (lhs: SheetAction, rhs: SheetAction) -> Bool {
```

Comment:
Is this custom Equatable implementation required over just using the automatically synthesized conformance?

Reply:
@staykids good point, I think that was a misplaced assumption of mine with enums like this. This is removed. Hope you don't mind but I noticed another custom implementation on `RemoteNotificationType`. I didn't notice any regressions after removing it so I added that to the commit as well.